src/localization.js

```diff
@@ -12,19 +12,26 @@ var language_mapping = {
   hu: require('../i18n/hu'),
   vi: require('../i18n/vi'),
   'pt-BR': require('../i18n/pt-BR'),
-  'zh-Hans': require('../i18n/zh-Hans')
 };
 
 module.exports = {
-  getLanguages: function() {
+  getLanguages: function () {
```

Comment:
```suggestion
  getLanguages: function() {
```

Plase, keep the same code formating as original code.